app/static/js/NeighbourPlotCanvas.js

```diff
@@ -274,7 +342,7 @@ NeighbourPlotCanvas.prototype._createNavigateBehaviour = function() {
     return d3.behavior.zoom()
         .x(this.xScale)
         .y(this.yScale)
-        .scaleExtent(config.scaleExtent)
+        .scaleExtent(config.zoomExtent)
```

Comment:
Is this an error from a previous PR? If so, is it feasible to write a test that catches it?